service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -47,6 +50,10 @@ public function startAction(): Response
         /** @var string $type */
         $type = $this->params()->fromQuery("personType");
 
+        if (! $this->lpaFormHelper->lpaIdentitiesMatch($lpas, $type)) {
+            $personTypeDescription = $type === 'donor' ? "Donors" : " Certificate Providers";
+            throw new OpgFrontException("These LPAs are for different " . $personTypeDescription);
```

Comment:
`HttpException` displays a proper error page that tells the user what the error message is (debug information isn't shown in prod)
```suggestion
            throw new HttpException(400, "These LPAs are for different " . $personTypeDescription);
```